src/realm/util/interprocess_condvar.cpp

```diff
@@ -81,6 +95,7 @@ void InterprocessCondVar::close() noexcept
 #endif
     // we don't do anything to the shared part, other CondVars may share it
     m_shared_part = nullptr;
+#endif
```

Comment:
Why is `m_shared_part = nullptr;` being moved inside the `#ifdef REALM_CONDVAR_EMULATION`?